shared/config_sv.lua

```diff
@@ -1,7 +1,12 @@
 Config = {
     Spawn = vec(-75.5010, -818.9702, 326.1753, 221.4246),
     SaveTime = 10000,
-    Accounts = { cash = 0, bank = 0, black = 0, tebex = 0  },
+    Accounts = { 
```

Comment:
Any reason of using brackets?

Reply:
It's out of habit, I can remove the strings if requested

Reply:
Yeah. Leave it like it was. At the moment there's no need for brackets.

Reply:
done